modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -1123,6 +1123,20 @@ public Array(T[] array)
                 Add(element);
         }
 
+        /// <summary>
+        /// Constructs a new <see cref="Array{T}"/> from the given items.
+        /// </summary>
+        /// <param name="span">The items to put in the new array.</param>
+        /// <returns>A new Godot Array.</returns>
+        public Array(ReadOnlySpan<T> span)
+        {
+            _underlyingArray = new Array();
+            _underlyingArray.Resize(span.Length);
```

Comment:
This `Resize` will grow the array, and then the `Add` below will append at the end. So with a span like `[1, 2, 3]`, the array will end up like `[0, 0, 0, 1, 2, 3]`. Other constructors don't use `Resize` here, so I suggest to just remove it.

```suggestion
```